examples/form_kitchen_sink/src/tests.rs

```diff
@@ -14,31 +14,29 @@ impl fmt::Display for FormOption {
     }
 }
 
-fn assert_form_eq(client: &Client, form_str: &str, expected: String) {
-    rocket::async_test(async move {
-        let mut res = client.post("/")
-            .header(ContentType::Form)
-            .body(form_str)
-            .dispatch().await;
-
-        assert_eq!(res.body_string().await, Some(expected));
-    })
+async fn assert_form_eq(client: &Client, form_str: &str, expected: String) {
+    let mut res = client.post("/")
+        .header(ContentType::Form)
+        .body(form_str)
+        .dispatch().await;
+
+    assert_eq!(res.body_string().await, Some(expected));
 }
 
-fn assert_valid_form(client: &Client, input: &FormInput<'_>) {
+async fn assert_valid_form(client: &Client, input: &FormInput<'_>) {
```

Comment:
It would be nice to make this and the other assert macros so that we don't have to call `.await` every time, instead calling it in the macro itself.

Reply:
Very possible - https://paste.rs/9HQ.diff. However we do lose the documentation on what the types are supposed to be, and I think a blocking `Client` will end up being the nicer way to simplify this.

Reply:
I'm okay with either approach. This should _probably_ be a macro, however, so test failures point to the right line.